eslint/rules/valid-constructors.js

```diff
@@ -1,38 +1,102 @@
-// Rules
```

Comment:
The rendering of the diff on this file isn't amazingly helpful - reviewing the [end result](https://github.com/guardian/cdk/blob/aa-update-custom-lint-rule/eslint/rules/valid-constructors.js) might be easier?